### DISCUSSION TITLE
The runner has received a shutdown signal. This can happen when the runner service is stopped, or a manually started runner is canceled.

### DISCUSSION THREAD

**Discussion by sensedata1** [actions/virtual-environments#8640] [General questions]
Hi there, our cdk web app deployment has just started failing with `The runner has received a shutdown signal. This can happen when the runner service is stopped, or a manually started runner is canceled.` The only error in the logs is `Error: The operation was canceled.` This happened once last week, and now deployments are failing consistently after a seemingly unrelated commit to increase the timeout on a golang linting operation. 
Is there any way to get more meaningful log output from the job? We're currently unable to deploy to production due to this failure.

The repo is private but here's an example of a failed run https://github.com/noah-labs/noah-main/actions/runs/6611401958/job/17955919782

**Comment by ilia-shipitsin** [comment]
that's a good question. I'm afraid it is beyond the scope of "runner-images" repo, but rather related to actions in general.
we redirected similar questions to our colleagues, as far as I remember, all such cases were due to resource exhaustion on runner (mem/CPU), however that telemetry is not exposed to customers.

**Comment by vpolikarpov-akvelon** [comment]
This problem is being discussed here: https://github.com/actions/runner-images/discussions/8640

**Comment by sensedata1** [comment]
For anyone else who arrives here, pinning the ubuntu runner to `ubuntu-20.04` has allowed the job to run.
